client/localsink/fullstate/fullstate.go

```diff
@@ -14,11 +14,13 @@ type ServiceEndpoints struct {
 }
 
 type Callback func(item <-chan *ServiceEndpoints)
+type Setup func()
 
 // EndpointsClient is a simple client to kube-proxy's Endpoints API.
 type Sink struct {
 	Config   *localsink.Config
 	Callback Callback
+	SetupFcn Setup
```

Comment:
I think renaming this to `SetupFunc` would be more go-style

Reply:
Done